content/miraclelanding/index.md

```diff
@@ -0,0 +1,52 @@
++++
+title = "Temakveld: Mirakellanding på Hudson-elva"
+lastmod = 2020-09-27T23:00:00
+[menu]
+main = { parent = "Annet" }
++++
+
+{{< img
+    class="img-fluid float-right"
+    src="landing.jpg"
+    alt=`Fly ligger i Hudson-elven med utslåtte nødutganger og mennesker på flymaskinen.`
+    >}}
+
+{{% ingress %}}
+
+I år er det 11 år siden Chesley Sullenberger og Jeffery Skiles ble tvunget til å lande en
+Airbus A320 i Hudson-elven i New York City. Velkommen på temakveld hvor vi tar opp tråden og
+ser en dokumentar om hendelsen.
+
+{{% /ingress %}}
+
+Hvis du ikke kjenner til hendelsen, les [US Airways Flight 1549 på
+Wikipedia](https://no.wikipedia.org/wiki/US_Airways_Flight_1549).
+
+##### Kveldens program
+
+- Inntak av autentisk mat fra New York City.  
+- Vi ser dokumentaren "Mirakellanding på Hudson-elven", tilgjengelig på Disney+.  
+- Kviss: "Hvordan skal vi oppføre oss i en ditchingsituasjon?"  
+
+##### Praktisk informasjon
+
+Dato og klokkeslett: Det blir selvfølgelig 15. januar 2020, kom etter 18.00.
```

Comment:
Gleder meg til å vende tilbake til tiden før pandemien. Kan du låne meg en tidsmaskin?

Reply:
Det er hyggelig at du synes det. Takk for at du påpekte dette. Det var jo dumt av meg å legge til et årstall - det er jo ingen grunn til å ikke arrangere dette hvert år fremover. Jeg skal endre det etterpå til "15. januar ♾".